app/models/metric/purging.rb

```diff
@@ -1,6 +1,6 @@
 module Metric::Purging
   def self.purge_date(type)
-    value = VMDB::Config.new("vmdb").config.fetch_path(:performance, :history, type.to_sym)
+    value = ::Settings.fetch_path(:performance, :history, type.to_sym)
```

Comment:
For this one you would do something like `value = ::Settings.performance.history[type]` (No need even for the to_sym anymore)